examples/black_scholes_greeks.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python
+
+# Copyright 2021-2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+from enum import IntEnum
+
+from benchmark import CuNumericTimer, parse_args, run_benchmark
+
+NUM_ITERS = 10
+WARMUP_ITER = 2
+
+vol_start = 0.1
+vol_step = 0.01
+t_start = 0.5
+t_step = 1.0 / (365 * 10)
+money_start = -0.4
+money_step = 0.01
+
+
+RISKFREE = 0.02
+S0 = 100.0
+N_GREEKS = 7
+
+
+class Greeks(IntEnum):
+    PREM = (0,)
+    DELTA = (1,)
+    VEGA = (2,)
+    GAMMA = (3,)
+    VANNA = (4,)
+    VOLGA = (5,)
+    THETA = 6
```

Comment:
What is the function of the 1-tuples here?
```
In [3]: class Greeks(IntEnum):
   ...:     PREM = (0,)
   ...:     DELTA = (1,)
   ...:     VEGA = (2,)
   ...:     GAMMA = (3,)
   ...:     VANNA = (4,)
   ...:     VOLGA = (5,)
   ...:     THETA = 6
   ...: 

In [4]: Greeks.PREM
Out[4]: <Greeks.PREM: 0>

In [5]: class Greeks(IntEnum):
   ...:     PREM = 0
   ...:     DELTA = 1
   ...:     VEGA = 2
   ...:     GAMMA = 3
   ...:     VANNA = 4
   ...:     VOLGA = 5
   ...:     THETA = 6
   ...: 

In [6]: Greeks.PREM
Out[6]: <Greeks.PREM: 0>
```

Reply:
There is no particular reason for this, removing 1-tuples